respy/simulate.py

```diff
@@ -159,43 +146,47 @@ def simulate(
     state_space = solve_with_backward_induction(state_space, optim_paras, options)
 
     # Prepare simulation.
+    n_simulation_periods = int(df.index.get_level_values("period").max() + 1)
```

Comment:
Because ``options["n_periods"]`` governs the time horizon for which the model is solved which can be different than the time horizon for the simulation.

As an example, I choose ``options["n_periods"] = 50`` such that agents behave like the maximize expected utility until retirement. I want to estimate parameters via MSM, but have only moments for the first 20 periods. Then, I could save some time simulating more periods than necessary.

Reply:
Makes sense!